client/src/translations/en.json

```diff
@@ -261,6 +261,7 @@
   "reselect_source_column": "Pick another column",
   "resources": "Resources",
   "request_permission_from_your_organisation_admin": "Request permission from your organisation admin.",
+  "retry": "Retry",
```

Comment:
es.json version should be
"retry": "Volver a intentar"
"visualise": "Visualizar"


for fr.json maybe @loicsans can help :)

Reply:
will do the french... give me a few minutes...